.github/workflows/deploy-prod.yml

```diff
@@ -20,16 +20,13 @@ jobs:
       ECS_SERVICE: keycloak-prod
 
     steps:
-      - uses: actions/checkout@v2
       - uses: mbta/actions/deploy-ecs@v1
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           ecs-cluster: ${{ env.ECS_CLUSTER }}
           ecs-service: ${{ env.ECS_SERVICE }}
-          # TODO once image is built locally, remove DOCKER_REPO and
-          # get this value from the build-push-ecr step
-          docker-tag: ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.docker_tag }}
+          docker-tag: ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.tag }}
```

Comment:
I'm not sure it's worth the effort; as currently written the prod workflow doesn't do any manipulation of container images, or even check out the code. I added the `dev` tag for Keycloak Dev deploys on a whim, but I'm not sure how useful it will prove to be.